packages/eslint/src/plugins/plugin.ts

```diff
@@ -153,6 +180,9 @@ function buildEslintTargets(
           isRootProject ? '{projectRoot}/' : '{projectRoot}'
         )
       ),
+      ...(existsSync(join(workspaceRoot, projectRoot, '.eslintignore'))
+        ? ['{projectRoot}/.eslintignore']
+        : []),
```

Comment:
This includes `proj/.eslintignore` as an input?

Should it also include anything in a parent directory.. as well as anything in a child directory?

Reply:
Only the `.eslintignore` file at the project root (cwd) applies (https://eslint.org/docs/latest/use/configure/ignore-deprecated#the-eslintignore-file). Other `.eslintignore` are not used by ESLint.